WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabViewController.swift

```diff
@@ -11,6 +11,7 @@ class ReaderTabViewController: UIViewController {
         return makeReaderTabView(viewModel)
     }()
 
+    private let settingsButton: SpotlightableButton = SpotlightableButton(type: .custom)
     private let searchButton: SpotlightableButton = SpotlightableButton(type: .custom)
```

Comment:
[nit] We can use a normal UIButton for searchButton now

Reply:
Done! 248ece3